### PR TITLE
tidies up simple animal sprites, adds movement states, improves some sprites.

### PR DESCRIPTION
**Pull request by alsoandanswer**
I took the crab and spider sprites from /tg/. Don't worry though, they still have their signature colors.
so, you know. you can tell which one will give you spider eggs.

There's also new movement states.

I've tidied up and deleted some unused files too, so it's easier to sprite for later.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by Arrow768** [comment]
It seems you removed the cavern.fmi file, but I dont see where the sadrone got readded.

File: code/modules/mob/living/simple_animal/friendly/adhomai.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /mob/living/simple_animal/ice_tunneler
 	name = "ice tunneler"
 	desc = "An egg producing beast from Adhomai. It is known for burrowing in ice and snow."
+	icon = 'icons/mob/npc/livestock.dmi'
```
Comment (Sindorman) [inline code/modules/mob/living/simple_animal/friendly/adhomai.dm]: This is messy. There are so many aniamals which can be united under `/livestock/` object class and have this assigned, to have cleaner code... but idk

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 2]
```diff
@@ -2,6 +2,7 @@
 /mob/living/simple_animal/cat
 	name = "cat"
 	desc = "A domesticated, feline pet. Has a tendency to adopt crewmembers."
+	icon = 'icons/mob/npc/pets.dmi'
```
Comment (Sindorman) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: same as livestock comment but `/pets/` class

**Comment by alsoandanswer** [comment]
> Keep the spider sprites as they are now. These seem like a downgrade.

They were too complex to add a moving animation for.

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**